support/controlplane-component/token-minter-container.go

```diff
@@ -116,17 +113,30 @@ func (opts TokenMinterContainerOptions) buildContainer(hcp *hyperv1.HostedContro
 			},
 		},
 		VolumeMounts: []corev1.VolumeMount{
-			{
-				Name:      kubeconfingVolumeName,
-				MountPath: kubeconfigMountPath,
-			},
 			{
 				Name:      tokenVolume.Name,
 				MountPath: tokenFileMountPath,
 			},
 		},
 	}
 
+	if opts.KubeconfigSecretName != "" {
```

Comment:
whats the rationale behind these changes?

Reply:
needed for the CPO, as it doesn't mount a kubeconfig volume, but passed a kubeconfig secret name to the TokenMinter